k8s/namespaces/admin/csi-secret-store-provider/kustomization.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - secrets-store-csi-driver.yaml
+  - https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/deployment/provider-azure-installer.yaml
```

Comment:
Must point to release tag than master as we don't want breaking changes applied .